Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM osrf/ros:humble-desktop-full
+
+# Install dependencies (taken from devcontainer dockerfile)
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
+    apt-get update -q && \
+    apt-get install -yq --no-install-recommends \
+    lcov \
+    curl \
+    wget \
+    python3-pip \
+    python-is-python3 \
+    python3-argcomplete \
+    python3-colcon-common-extensions \
+    python3-colcon-mixin \
+    python3-rosdep \
+    libpython3-dev \
+    python3-vcstool \
+    ros-humble-tl-expected && \
+    rm -rf /var/lib/apt/lists/*
+
+# Create ROS workspace
+WORKDIR /ros_ws/src
+
+# Clone driver code
+RUN git clone https://github.com/bdaiinstitute/spot_ros2.git .
+
+# Change gitmodule links to HTTPS to anonymously clone them
+RUN sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
```

Comment:
@ryan-roche meta: which `ssh` URLs? I see none on [`main`](https://github.com/bdaiinstitute/spot_ros2/blob/main/.gitmodules).

Reply:
I think I had this confused for the ROS1 driver repository...

Resolved by 661e364